errors/src/main/java/com/palantir/conjure/java/api/errors/QosException.java

```diff
@@ -61,13 +69,24 @@ public static Throttle throttle() {
         return new Throttle(Optional.empty());
     }
 
+    /**
+     * Returns stuff.
+     */
+    public static Throttle throttle(QosReason reason) {
+        return new Throttle(Optional.empty(), reason);
+    }
+
     /**
      * Like {@link #throttle()}, but includes a cause.
      */
     public static Throttle throttle(Throwable cause) {
         return new Throttle(Optional.empty(), cause);
     }
 
+    public static Throttle throttle(Throwable cause, QosReason reason) {
```

Comment:
Let's order the reason first -- it's common for causal throwables to be the final argument.